include/deal.II/lac/arpack_solver.h

```diff
@@ -76,6 +76,14 @@ extern "C" void dneupd_(int *rvec, char *howmany, int *select, double *d,
  * <code>dnaupd</code> work and also how to set the parameters appropriately
  * please take a look into the ARPACK manual.
  *
+ * @note Whenever you eliminate degrees of freedom using
+ * ConstraintMatrix, you generate spurious eigenvalues and
+ * eigenvectors. If you make sure that the diagonals of eliminated
+ * matrix rows are all equal to one, you get a single additional
+ * eigenvalue. But beware that some functions in deal.II set these
+ * diagonals to rather arbitrary (from the point of view of
+ * eigenvalue problems) values.
```

Comment:
Yes, but maybe in a separate pull request? I think most of the text simply explains where the problem comes from, and that should stay the same. But what to do about the problem can certainly change.

I think you probably still want to reference step-36 here. (Also, did you want to remove the author attribution?)